source/code/plugins/patch_management_lib.rb

```diff
@@ -278,9 +274,9 @@ def transform_and_wrap(
                     "OSFullName" => osFullName,
                     "Collections"=> collections
                 }]}
-          @log.debug "LinuxUpdates : installedPackages x #{installedPackages.size}, 
-                                        availableUpdates x #{availableUpdates.size}"
+          OMS::Log.info_once("LinuxUpdates : installedPackages x #{installedPackages.size}, 
```

Comment:
is it not better to leave it as debug as it will help in debugging to log it continously

Reply:
discussed with Koshy and Varad, its clarified that it would of info once and the message will be in debug after that.